packages/editor/src/client/routes/articleList.tsx

```diff
@@ -76,6 +78,18 @@ export function ArticleList() {
     refetch(articleListVariables)
   }, [filter, page, limit, sortOrder, sortField])
 
+  const [duplicatedDataId, setDuplicatedDataId]: any = useState()
+  let output;
```

Comment:
Why do you initialize `output` here? You only need it where you actually duplicate the article.